qa/workunits/rbd/concurrent.sh

```diff
@@ -78,8 +78,13 @@ function setup() {
 
 	# This assumes it's easier to read a file than generate
 	# random data.  Use busybox because it is a big executable.
-	dd if="/bin/busybox" of="{SOURCE_DATA}" bs=2048 count=66 \
-		>/dev/null 2>&1
+        if [ -f /bin/busybox ]; then
+            dd if="/bin/busybox" of=${SOURCE_DATA} bs=2048 count=66 \
+                >/dev/null 2>&1
+        else
+            dd if=/dev/urandom of=${SOURCE_DATA} bs=2048 count=66 \
+               >/dev/null 2>&1
+        fi
 
```

Comment:
Wow, this is one properly broken test.  If it had worked with that conf stuff it would have been comparing a zero length file against what essentially is a /dev/zero and then suppressing the "hit EOF" result it looks like.

Drop this busybox stuff altogether, using /dev/urandom for ~140K is OK.  No need to do this in two commits - squash everything into one commit and call it "qa: unbreak concurrent.sh workunit" or similar.